pkg/k8s/cell_linux.go

```diff
@@ -54,6 +55,13 @@ var Cell = cell.Module(
 		}
 	}),
 
+	cell.Provide(func() endpointmanager.EndpointManagerConfig {
+		return endpointmanager.EndpointManagerConfig{
+			EndpointRegenInterval: time.Duration(0),
```

Comment:
https://github.com/cilium/cilium/pull/35815/files#diff-90b7c7e2b86aba14a07baaffbb3b8052b0ed20baaafe5667d2f26b1e7aa67990R28
confirmed that setting this to 0 will disable this